xbmc/addons/kodi-addon-dev-kit/include/kodi/libXBMC_addon.h

```diff
@@ -203,14 +203,17 @@ namespace ADDON
      * @brief Add a message to XBMC's log.
      * @param loglevel The log level of the message.
      * @param format The format of the message to pass to XBMC.
+     * @note This method uses limited buffer (16k) for the formatted output.
+     * So data, which will not fit into it, will be silently discarded.
      */
     void Log(const addon_log_t loglevel, const char *format, ... )
     {
       char buffer[16384];
       va_list args;
       va_start (args, format);
-      vsprintf (buffer, format, args);
+      vsnprintf (buffer, sizeof (buffer), format, args);
       va_end (args);
+      buffer[sizeof (buffer) - 1] = '\0'; // to be sure it's nul-terminated
```

Comment:
typo: `null-terminated`

Reply:
I'm referring here to nul ASCII character, https://tools.ietf.org/html/rfc20#section-2, not to null pointer.

But if you wish, I can change it. Should I?

Reply:
Most (all?) documentation call it null termination, but you can keep it as it is.